root/backend/src/server.ts

```diff
@@ -1,18 +1,27 @@
 import express from 'express';
 import cors from 'cors';
-
+import mongoose from 'mongoose';
 //  routes
 import weatherAPI from './routes/weatherAPI';
 
+require('dotenv/config');
+
 const app: express.Application = express();
+const key: string = process.env.WEATHER_API_KEY!;
```

Comment:
Is this supposed to be `...WEATHER_API_KEY`?

Reply:
nope, should be fixed now